### ISSUE TITLE
confirmPlatformPayPayment not return object for google pay.

### ISSUE THREAD

**Issue by AjayGol** [stripe/stripe-react-native#1366]
**confirmPlatformPayPayment using for google pay**
```
 const { error, paymentMethod } = await confirmPlatformPayPayment(
          clientSecret,
          {
            googlePay: {
              testEnv: true,
              merchantName: '***',
              merchantCountryCode: 'US',
              currencyCode: 'USD',
              billingAddressConfig: {
                format: PlatformPay.BillingAddressFormat.Full,
                isPhoneNumberRequired: false,
                isRequired: false,
              },
            },
          },
        );
```

**To Reproduce**
Using simple demo code. Modal is open and payment made success(Check stripe log and payment success). But method not return object. - Not return  { error, paymentMethod }  value.

**Expected behavior**
Must be return object error or paymentMethod after close modal.

**Desktop (please complete the following information):**
 - Android
 - Google PAY
 - 0.26.0

Please help me what's reason for not return object.

**Comment by charliecruzan-stripe** [comment]
hi @AjayGol , could you provide more information on how to reproduce this? I can't trigger this same behavior on my end.

**Comment by AjayGol** [comment]
Here is more detail.

Android configuration
```
{
        buildToolsVersion = "29.0.2"
        minSdkVersion = 21
        compileSdkVersion = 33
        targetSdkVersion = 33
        ndkVersion = "25.1.8937393"
        androidXAnnotation = "1.1.0"
        androidXBrowser = "1.0.0"
        kotlinVersion = "1.8.0"
    }
```


React-Native version - 0.64.2

I have get done transition in staging more and project both mode. But After transition I have not get response.


Example
```
const { error, paymentMethod } = await confirmPlatformPayPayment(
          clientSecret,
          {
            googlePay: {
              testEnv: false,
              merchantName: '****',
              merchantCountryCode: 'US',
              currencyCode: currency,
            },
          },
        );
 if (error) {
          alert(error?.message);
          // Update UI to prompt user to retry payment (and possibly another payment method)
        } else {
          alert('Success', 'The payment was confirmed successfully.');
        }        
        
```

After transition no any alert coming. after confirmPlatformPayPayment method code never execute.

**Comment by charliecruzan-stripe** [comment]
@AjayGol I have a feeling this might be related to RN being on version 0.64.2. Have you tried on 0.65 or higher?

So far I haven't been able to reproduce this

**Comment by AjayGol** [comment]
@charliecruzan-stripe let me update RN version and let you know. At your end 0.65 version is working fine so I will update this version only. (Because if I will update latest version then it may be take lot time). Just confirm 0.65 is working or not so this is very helpfully.

**Comment by charliecruzan-stripe** [comment]
It's working in our example app that uses 0.69.7, but 0.65 has significant differences to 0.64 so I'd recommend trying that out

**Comment by AjayGol** [comment]
@charliecruzan-stripe Let me update to 0.69.7. Package need work in lower react-native version as well. Let me update 0.69.7 and let you know.

Thanks

**Comment by charliecruzan-stripe** [comment]
@AjayGol any update so far?

**Comment by charliecruzan-stripe** [comment]
Closing for inactivity

**Comment by annx10kn** [comment]
This problem resolved by upgrade react native to 0.66.0 (my previous version is 0.64.3)
